Makefile

```diff
@@ -1,20 +1,19 @@
-.PHONY: ui vendor build container-tornjak-backend-spire container-tornjak-backend-spire-push container-manager container-manager-push release-tornjak-backend-spire-multiversions push container-frontend container-frontend-push container-tornjak-backend container-tornjak-backend-push
+.PHONY: ui vendor build container-tornjak-backend-spire container-manager container-manager-push push container-frontend container-frontend-push container-tornjak-backend container-tornjak-backend-push
 
 VERSION=$(shell cat version.txt)
 
+## when containers are built, they are tagged with these
 CONTAINER_TAG ?= tsidentity/tornjak:$(VERSION)
```

Comment:
I think the confusion starts here, with the variable names. 
Perhaps we should have the following naming convention:
```
CONTAINER_BACKEND_IMAGEPATH_TEST ?= tsidentity/tornjak-backend
CONTAINER_BACKEND_IMAGEPATH ?= ghcr.io/spiffe/tornjak-backend
```

And then we use (and tag) them accordingly?